ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/representative-journey/pages/about-your-client/about-your-client.component.html

```diff
@@ -47,11 +47,11 @@ <h1 class="govuk-heading-xl" i18n="@@aboutYourClient_h1">
                       <span *ngIf="isTextInputInvalid" class="govuk-error-message" i18n="@@aboutYourClient_span_2">
                         Please complete this section
                       </span>
-                      <label class="govuk-label" i18n="@@aboutYourClient_label_1">
+                      <label class="govuk-label" id="more-detail-hint" i18n="@@aboutYourClient_label_1">
                         Please provide more information
                       </label>
                     </div>
-                    <textarea formControlName="textInput" aria-label="details" class="txtMore vh-text-width"
+                    <textarea formControlName="textInput" aria-label="more-detail-hint" class="txtMore vh-text-width"
```

Comment:
https://developer.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/

`aria-labelledby`